assets/stylesheets/main.css

```diff
@@ -55,3 +55,81 @@
 .chat-from-self {
   font-weight: bold;
 }
+
+/* (NEW): Common stylings for modal */
+.modal-container {
+  animation: modalContainerFade ease 0.1s;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+  z-index: 1;
+}
+
+@keyframes modalContainerFade {
+  0% {
+    background-color: rgba(0, 0, 0, 0);
+  }
+  100% {
+    background-color: rgba(0, 0, 0, 0.3);
+   }
+}
+
+.modal {
+  animation: modalFade ease 0.1s;
+  background-color: white;
+  justify-content: space-between;
+  position: relative;
+  top: 50%;
+  transform: translateY(-70%);
+  margin-left: auto;
+  margin-right: auto;
+  width: 70%;
+  height: 50%;
+  border: 5px solid;
+  border-radius: 20px;
+}
+
+@keyframes modalFade {
+  0% {
+   opacity: 0;
+  }
+  100% {
+    opacity: 100%;
+   }
+}
+
+.close-modal-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  background-color: #e04141;
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.close-modal-button::after {
+  font-size: 20px;
+  color: white;
+  content: "\00d7"
```

Comment:
If it's okay, I might actually keep this one since unlike a triangle, I don't actually know how to draw a `X` in CSS. I also tried looking this up, but there were no solutions that were that simple, so I'm leaning towards just keeping this if possible.